test/vanilla/Expected/AcceptanceTests/BodyString/bodystring/aio/operations_async/_string_operations_async.py

```diff
@@ -452,15 +452,15 @@ async def get_not_provided(
     async def get_base64_encoded(
         self,
         **kwargs
-    ) -> bytearray:
+    ) -> bytes:
```

Comment:
These changes from bytearray to bytes are due to [this](https://github.com/Azure/autorest.testserver/commit/70c108c95aa21b934b97278c67aedc6ae22e1588) testserver change